tests/foreman/ui/test_provisioningtemplate.py

```diff
@@ -37,8 +37,27 @@ def template_data():
     return read_data_file(OS_TEMPLATE_DATA_FILE)
 
 
+@fixture(scope='function', autouse=False)
+def clone_setup(module_org, module_loc):
+    name = gen_string('alpha')
+    content = gen_string('alpha')
+    os_list = [
+        entities.OperatingSystem().create().title for _ in range(2)
+    ]
+    return {
+        'pt': entities.ProvisioningTemplate(
+            name=name,
+            organization=[module_org],
+            location=[module_loc],
+            template=content,
+            snippet=False
+            ).create(),
+        'os': os_list
```

Comment:
Can we rename this key to `os_list`? `os` suggest singular, while value is list.

Reply:
you serious? :D ok, i'll do it